app/src/main/java/simform/gitexcercise/android/ui/auth/activity/ForgetPasswordActivity.kt

```diff
@@ -0,0 +1,16 @@
+package simform.gitexcercise.android.ui.auth.activity
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import simform.gitexcercise.android.databinding.ActivityForgetPasswordBinding
+
+class ForgetPasswordActivity : AppCompatActivity() {
```

Comment:
Name should be "ForgotPasswordActivity"..replace it everywhere.

Reply:
Resolved.